src/lua/bcc/vendor/posix.lua

```diff
@@ -62,9 +64,15 @@ local function cpu_count()
   return tonumber(ffi.C.get_nprocs())
 end
 
+local function tonumber64(n, base)
+  assert(type(n) == "string")
+  return ffi.C.strtoull(n, nil, base or 10)
```

Comment:
Would you want `base or 0` to invoke strtoull's automatic base logic? Although so far everybody is calling it with base=16.